md/release-notes.md

```diff
@@ -83,6 +83,10 @@ When defining actors in a pool, it is now possible to remove the "initiator" tag
 ### UI Designer properties autocomplete for variables
 Autocomplete capacity has been added in UI Designer properties fields, to help you input variable names with no risk of typos anymore.
 
+### UI Designer allows handling collections with invalid state 
+Collection with invalid content now keeps the model available allowing modification on the structure of the collect (for instance add and remove). 
```

Comment:
Is it invalid content or invalid state? Article title says "state", article content says "content".
How can Fred get in this situation of invalid "state/content"? It this obvious for anyone that has not encountered the issue? If not, this should be stated.
What was the problem with the fact that the model was discarded? Fred had to create it all again or? 
The "allowing..." part of the sentence should be made into another sentence => First sentence says what has been done, second sentence talks about the value.
What is a "collect"? Collection? If so, it needs to get all its letters.